frontend/src/config.js

```diff
@@ -1,2 +1,2 @@
 export const DOMAIN_URL = 'http://localhost';
-export const API_URL = 'http://localhost';
+export const API_URL = '';
```

Comment:
I don't understand why are you removing the API_URL?

Reply:
By hosting static files on nginx, it is not necessary.